requirements.yml

```diff
@@ -53,6 +53,10 @@
   src: https://github.com/open-craft/ansible-load-balancer
   version: origin/master
 
+- name: consul
+  src: https://github.com/open-craft/ansible-role-consul
+  version: uman/consul
```

Comment:
I always had to use `origin/uman/consul` in the past, but maybe the newer version of Ansible we are using now finally allows to omit the `origin/` part.  In any case, we will need to update this to the master branch before merging.

Reply:
Yeah somehow it managed to work, I also remember having to prepend `origin/`.